web/src/copy_and_paste.ts

```diff
@@ -472,6 +489,122 @@ export function paste_handler_converter(paste_html: string): string {
             return prefix + content + (node.nextSibling && !content.endsWith("\n") ? "\n" : "");
         },
     });
+    const processed_math_block_parents = new Set();
+    const processed_inline_math_block_parents = new Set();
+    turndownService.addRule("katex-math-block", {
+        filter(node: Node) {
+            if (!(node instanceof HTMLElement)) {
+                return false;
+            }
+            // We only need nodes whose parent contains the katex-display spans as immediate children or katex spans as immediate children
```

Comment:
This comment is useless; explain instead why this logic is the correct way to detect katex nodes.